src/main/kotlin/me/nobaboy/nobaaddons/features/fishing/SeaCreatureType.kt

```diff
@@ -0,0 +1,18 @@
+package me.nobaboy.nobaaddons.features.fishing
+
+import dev.isxander.yacl3.api.NameableEnum
+import me.nobaboy.nobaaddons.utils.tr
+
+enum class SeaCreatureType : NameableEnum {
+	RARITY,
+	HOTSPOT,
+	RARE,
+	HOTSPOT_RARE;
+
+	override fun getDisplayName() = when(this) {
+		RARITY -> tr("nobaaddons.label.seaCreatureType.rarity", "Rarity")
+		HOTSPOT -> tr("nobaaddons.label.enchantDisplayMode.hotspot", "Hotspot")
+		RARE -> tr("nobaaddons.label.enchantDisplayMode.rare", "Rare")
+		HOTSPOT_RARE -> tr("nobaaddons.label.enchantDisplayMode.hotspotRare", "Hotspot & Rare")
```

Comment:
```suggestion
		RARE -> tr("nobaaddons.label.enchantDisplayMode.rare", "Rare Creatures")
		HOTSPOT_RARE -> tr("nobaaddons.label.enchantDisplayMode.hotspotRare", "Hotspot & Rare Creatures")
```

just saying 'Rare' in an option that also has 'Rarity' is a bit too ambiguous

rare creatures should also be explained further in the option description to explain how it differs from rarity